yaks/README.md

```diff
@@ -665,11 +678,42 @@ yaks.call(array_of_widgets, mapper: MyCollectionMapper, item_mapper: WidgetMappe
 ```
 
 If the mapper is left unspecified, Yaks will inspect whatever you pass
-it, and try various constant lookups based on naming. These all happen
+it. First it will test the given object against the mapping defined using `mapper_for`.
+If no mapper is found, it wil call `derive_mapper_from_item` or `derive_mapper_from_collection`
+depending on whether the given object is a collection or not. If the object responds
+to `to_ary` it is considered a collection.
+
+Notes:
+* you may override one of these two methods independently from the other.
+* this function takes either a symbol or any instance
+
+### mapper_for
+
+This method allows you to define one-to-one mapping between a class
+and its mapper class. For example, when using `mapper_for(Post, SpecialMapper)`,
+Yaks would always derive `SpecialMapper` from an instance of `Post`.
+
+Mapping define this way will have the highest priority during lookup.
+
+This methods first parameter can either be a symbol, a class or a lambda as followed:
+```ruby
+yaks = Yaks.new do
+  mapper_for(:comment, CommentMapper)
```

Comment:
This can be a bit misleading, as it looks like you'll be rendering a `Comment`. Maybe 

```ruby
mapper_for(:home, HomeMapper)
```

would be better